frontend/components/icons/phone-icon.tsx

```
@@ -0,0 +1,72 @@
+                <path
+                    d='M16.2144 7.13619C17.8758 7.41479 19.3849 8.19992 20.5897 9.40294C21.7945 10.606 22.5744 12.1129 22.8597 13.7718C22.9295 14.1897 23.2909 14.481 23.7031 14.481C23.7538 14.481 23.7982 14.4746 23.8489 14.4683C24.3182 14.3923 24.6289 13.9491 24.5528 13.4806C24.2104 11.4734 23.2592 9.64355 21.8071 8.19359C20.3551 6.74363 18.5225 5.79387 16.5125 5.45196C16.0432 5.37598 15.6057 5.68623 15.5233 6.14845C15.4409 6.61066 15.7452 7.06021 16.2144 7.13619Z'
+                    fill='white'
+                />
+                <path
+                    d='M29.9869 13.2336C29.4226 9.92848 27.8627 6.92092 25.4659 4.52754C23.069 2.13415 20.0571 0.576553 16.7471 0.0130316C16.2842 -0.0692806 15.8467 0.247305 15.7642 0.709519C15.6882 1.17807 15.9989 1.61495 16.4681 1.69727C19.423 2.19747 22.1179 3.59678 24.2611 5.73056C26.4043 7.87068 27.7993 10.5617 28.3003 13.5122C28.37 13.9301 28.7314 14.2214 29.1436 14.2214C29.1943 14.2214 29.2387 14.215 29.2894 14.2087C29.7523 14.1391 30.0694 13.6959 29.9869 13.2336Z'
+                    fill='white'
+                />
+            </g>
+            <defs>
+                <clipPath id='clip0_0_267'>
+                    <rect width='30' height='30' fill='white'></rect>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)